warehouse/config.py

```diff
@@ -304,15 +318,15 @@ def configure(settings=None):
     )
     maybe_set(settings, "warehouse.downloads_table", "WAREHOUSE_DOWNLOADS_TABLE")
     maybe_set(settings, "celery.broker_url", "BROKER_URL")
-    maybe_set(settings, "celery.result_url", "REDIS_URL")
-    maybe_set(settings, "celery.scheduler_url", "REDIS_URL")
-    maybe_set(settings, "oidc.jwk_cache_url", "REDIS_URL")
+    maybe_set_redis(settings, "celery.result_url", "REDIS_URL", db=12)
+    maybe_set_redis(settings, "celery.scheduler_url", "REDIS_URL", db=0)
+    maybe_set_redis(settings, "oidc.jwk_cache_url", "REDIS_URL", db=1)
```

Comment:
This will reset our cache state for JWKs in Trusted Publishers, once.

Reply:
I _think_ this is the code that will auto-fill the cache on the next invocation, so probably pretty safe.
https://github.com/pypi/warehouse/blob/33049d18e3c37d7b09644b11b0f1e65582deed56/warehouse/oidc/services.py#L215-L217

Reply:
aye it will.